templates/terraform-module-page.md

```diff
@@ -0,0 +1,85 @@
+{{- defineDatasource "config" .Env.README_YAML | regexp.Replace ".*" "" -}}
+{{- defineDatasource "includes" .Env.README_INCLUDES | regexp.Replace ".*" "" -}}
+---
+title: {{ (ds "config").name }}
+{{- if has (ds "config") "description" }}
+description: |-
+{{ (ds "config").description | strings.Indent 2 }}
+{{- end }}
+{{- if has (ds "config") "categories" }}
+categories:
```

Comment:
I tried doing this for `faq` and it's not working. Can you confirm? (screenshot)

Reply:
README.yaml
```yaml
tags:
 - aws

categories:
  - faq
```

![image](https://user-images.githubusercontent.com/19958900/49015237-8d6cbb80-f193-11e8-9262-88bb5bf5808e.png)
![image](https://user-images.githubusercontent.com/19958900/49015250-952c6000-f193-11e8-9301-a367cecaec60.png)